x/did/internal/secp256k1util/secp256k1util.go

```diff
@@ -0,0 +1,40 @@
+package secp256k1util
+
+import (
+	"fmt"
+
+	"github.com/btcsuite/btcutil/base58"
+	"github.com/tendermint/tendermint/crypto/secp256k1"
+)
+
+// PrivKeyFromBytes converts a byte slice into a Secp256k1 private key.
+// It returns an error when the length of the input is invalid.
+func PrivKeyFromBytes(bz []byte) (secp256k1.PrivKey, error) {
+	key := make([]byte, secp256k1.PrivKeySize)
```

Comment:
Before code : `var key secp256k1.PubKeySecp256k1`
But secp256k1.PubKeySecp256k1 is changed to secp256k1.PubKey.
`secp256k1.PubKey` type is only byte array(`[]byte`)
So, Changed to check key length code